cypher/cypher-docs/src/docs/dev/query-use.adoc

```diff
@@ -18,6 +18,9 @@ The `USE` clause determines which graph a query, or query part, is executed agai
 The `USE` clause determines which graph a query, or query part, is executed against.
 It is supported for queries and schema commands.
 
+[NOTE]
+The `USE` clause cannot be used together with the <<load-csv-importing-large-amounts-of-data, `PERIODIC COMMIT` clause>>.
```

Comment:
```
<<load-csv-importing-large-amounts-of-data, `PERIODIC COMMIT` clause>>
```
Having the backticks on `PERIODIC COMMIT` makes it look like the link is just the word `clause`. Would it be better to not have backticks on `PERIODIC COMMIT` so the whole thing looks like a link or keep it as it is?

Reply:
I've tried building it without the backticks, and I think it's best as-is.